erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -92,6 +92,7 @@ const BaseStack = (specInput) => {
       localDesc.sdp = that.enableSimulcast(localDesc.sdp);
     }
     localSdp = SemanticSdp.SDPInfo.processString(localDesc.sdp);
+    SdpHelpers.forceDirection(localSdp, isSubscribe);
```

Comment:
I think this code should only be added to the Firefox stack.

Reply:
Ok, I don't think could harm to force here too but if you prefer I will move to Firefox Stack

Reply:
Also they replied to me that Firefox now support transceivers, so we have to set direction in videoTransceiver and audioTransceiver. I think it's more an adapter job since they already do this in safari. So this will be moved in firefox stack as a temporary fix until adapter came out with their own solution